timemachine/fe/free_energy.py

```diff
@@ -284,7 +284,7 @@ def extract_trajectories_by_replica(self, atom_idxs: NDArray) -> NDArray:
         # (states, frames, atoms, 3)
         trajs_by_state = np.array(
             [
-                np.concatenate([np.array(chunk)[:, atom_idxs] for chunk in state_traj.frames._chunks()], axis=0)
+                np.concatenate([chunk[:, atom_idxs] for chunk in state_traj.frames._chunks()], axis=0)
```

Comment:
Ah, that's a good question. In this case I don't think we keep the full chunk in memory, because "advanced indexing" always returns a copy (https://numpy.org/doc/stable/user/basics.indexing.html#advanced-indexing).

```
>>> a = np.arange(100)
>>> b = a[0:-1:5]
>>> b.base is a
True
>>> idxs = np.arange(0, 100, 5)
>>> c = a[idxs]
>>> c.base is a
False
```

Might still be good to make a copy here anyway; seems like the small extra cost would probably be worth the additional clarity.

Reply:
Opted to instead add a note on the use of advanced indexing in https://github.com/proteneer/timemachine/pull/1329/commits/604e556d92594d4711141ba6ef44303516c0459d